defaults/main.yml

```diff
@@ -5,5 +5,5 @@ windows_exporter_arch: 'amd64'
 windows_exporter_download_url: https://github.com/prometheus-community/windows_exporter/releases/download/v{{ windows_exporter_version }}/windows_exporter-{{ windows_exporter_version }}-{{ windows_exporter_arch }}.msi
 windows_exporter_options: ''
 windows_exporter_bin_path: '"C:\Program Files\windows_exporter\windows_exporter.exe"'
-windows_exporter_state: restarted
-windows_exporter_enabled: true
+windows_exporter_state: started
```

Comment:
I switched the default state to started so that the role becomes idempotent. With restarted every time I run a play with this role, my services get restarted which is not desirable during normal operation imo.